.github/workflows/test.yaml

```diff
@@ -118,20 +120,20 @@ jobs:
         idp: [auth0, azure, github, gitlab, google, oidc, okta, onelogin, ping]
     runs-on: ${{ matrix.platform }}
     steps:
-      - uses: actions/setup-go@84cbf8094393cdc5fe1fe1671ff2647332956b1a # pin@v2
+      - uses: actions/setup-go@84cbf8094393cdc5fe1fe1671ff2647332956b1a
```

Comment:
i think the linter gobbled the comments

Reply:
I removed the comments. They weren't accurate.